sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html

```diff
@@ -14,28 +14,58 @@
 
     <!-- CSS: implied media=all -->
     <!-- CSS concatenated and minified via ant build script-->
-    <link rel="stylesheet" href="https://www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.1/libraries/foundation-6/css/foundation.css" type="text/css" media="all">
-    <link rel="stylesheet" href="https://www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.1/css/ebi-global.css" type="text/css" media="all">
-    <link rel="stylesheet" href="https://www.ebi.ac.uk/web_guidelines/EBI-Icon-fonts/v1.1/fonts.css" type="text/css" media="all">
+    <link rel="stylesheet" href="https://www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.2/libraries/foundation-6/css/foundation.css" type="text/css" media="all">
+    <link rel="stylesheet" href="https://www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.2/css/ebi-global.css" type="text/css" media="all">
+    <link rel="stylesheet" href="https://www.ebi.ac.uk/web_guidelines/EBI-Icon-fonts/v1.2/fonts.css" type="text/css" media="all">
 
     <!-- Use this CSS file for any custom styling -->
     <link rel="stylesheet" href="https://www.ebi.ac.uk/gxa/resources/css/foundation/atlas.css" type="text/css" media="all">
 
-    <script src="../dist/vendorCommons.bundle.js"></script>
-    <script src="../dist/experimentPage.bundle.js"></script>
+    <script src="/gxa_sc/vendorCommons.bundle.js"></script>
+    <script src="/gxa_sc/experimentPage.bundle.js"></script>
 </head>
 
 <body>
-    <div id="container"></div>
+    <div id="content" class="row column">
+      <div id="container"></div>
+    </div>
 </body>
 
 <script>
+    var tabs = [
+      {
+        type: 't-sne-plot',
+        name: 'Results',
+        props: {
+          ks: [2, 3, 4, 5, 6, 7, 8, 9],
+          perplexities: [1, 2, 3, 4, 5, 6],
+          suggesterEndpoint: 'json/suggestions'
+        }
+      },
+      {
+        type: 'experiment-design',
+        name: 'Experiment Design',
+        props: {}
+      },
+      {
+        type: '',
+        name: 'Supplementary Information',
+        props: {}
+      },
+      {
+        type: '',
+        name: 'Downloads',
+        props: {}
+      }
+    ]
+
```

Comment:
Just one question, are we going to have the same tabs as in gxa?

Reply:
I think the code for the tabs got copy pasted but probably the actual tabs will be different (maybe there won't be "Supplementary Information" or whatever)

Reply:
I guess that *Supplementary information* and *Experiment design* will be there, but the content will be differ substantially. At thte moment we have some empty placeholders. The current system of sending the tabs from the server and let the client side do the rendering is pretty flexible. This is what we agreed in the [Pivotal ticket](https://www.pivotaltracker.com/story/show/147753699) to have something very minimal.

Reply:
I guess that *Supplementary information* and *Experiment design* will be there, but the content will be differ substantially. At thte moment we have some empty placeholders. The current system of sending the tabs from the server and let the client side do the rendering is pretty flexible. This is what we agreed in the [Pivotal ticket](https://www.pivotaltracker.com/story/show/147753699) in order to have a minimal working implementation.

Reply:
You could even add the single-cell-specific tabs to the experiment page code (if you need rendering for all the other components and it's faster to reuse) and keep the same architecture.

Reply:
👍